client/lib/wp/sync-handler/index.js

```diff
@@ -51,6 +91,7 @@ export class SyncHandler {
 			// create a copy of the request params
 			const reqParams = Object.assign( {}, params );
 			const { path } = reqParams;
+			let localResponseBody = null;
```

Comment:
If this turns out to be too much work, I'm fine with having this. We can also update that in a future PR .